ReactiveObjC/RACNeverSignal.m

```diff
@@ -43,10 +43,10 @@ - (nullable RACDisposable *)subscribe:(id<RACSubscriber>)subscriber {
 
   RACCompoundDisposable *disposable = [RACCompoundDisposable compoundDisposable];
 
-  @rac_weakify(subscribers, subscriber);
+  @rac_weakify(subscribers);
   [disposable addDisposable:[RACDisposable disposableWithBlock:^{
-    @rac_strongify(subscribers, subscriber);
-    if (!subscribers || !subscriber) {
+    @rac_strongify(subscribers);
```

Comment:
I meant to retain `subscribers`, not `subscriber` (`subscriber` will be retained by being added to `subscribers`)
i.e `@rac_weakify(subscriber);`, `@rac_strongify(subscriber);`

But from looking on `RACCompoundDisposable`, when calling `dispose` the inner disposables get detached and therefor not retained (but someone please correct me here if I'm wrong). 

Therefore I agree with Barak - we can just create a cycle of:
subscriber -> compound disposable -> inner disposables array -> the subscriber disposable -> subscriber. That would get broken when calling dispose on the compound disposable.

```objc
[disposable addDisposable:[RACDisposable disposableWithBlock:^{
  id<RACSubscriber> __unused retainedSubscriber = subscriber;
}]];
```

Reply:
Agreed, simplified!